Python/challenges/day_3.py

```diff
@@ -0,0 +1,43 @@
+def create_item_dict():
+    item_dict = {}
+    for i in range(26):
+        letter = chr(i + ord("a"))
+        item_dict[letter] = [0, i + 1]
+    for j in range(26):
+        letter = chr(j + ord("A"))
+        item_dict[letter] = [0, j + 1 + 26]
+    return item_dict
+        
+def get_common_item(comps: list):
+    comp1 = comps.pop()
+    for i in range(len(comp1)):
+        item_found_in_all = True
+        for comp in comps:
+            if comp1[i] not in comp:
+                item_found_in_all = False
```

Comment:
A performance issue with this loop is that duplicate characters in `comp1` are checked again. For example, if its value is `aaaaab`.

Reply:
Could be solved with a list saving all used characters indeed